node/peer.js

```diff
@@ -460,7 +409,6 @@ PreferOutgoingHandler.prototype.getQOS = function getQOS() {
     }
 };
 
-// Consulted depending on the peer state
```

Comment:
I talked to @kriskowal. The bug is not in this PR, so I will create a different PR to fix it.